.github/workflows/ipa-renewal-test.yml

```diff
@@ -187,7 +188,7 @@ jobs:
           docker exec ipa pki nss-cert-show ipa-ca-agent
 
           # CA admin should be able to access PKI users
-          docker exec ipa pki -n ipa-ca-agent ca-user-find
+          docker exec ipa pki -n ipa-ca-agent --skip-revocation-check ca-user-find
```

Comment:
I think we should use `--ignore-cert-status EXPIRED_CERTIFICATE` since we're trying to fix an expired cert. If the cert is actually revoked we do want the client to know that (i.e. the command should fail).